README.md

```diff
@@ -142,26 +118,18 @@ publishing steps into one workflow.
 
 Ask @chriddyp to get NPM / PyPi package publishing access.
 
-1. Preparing to publish to NPM
-
-        # Bump the package version
-        $ npm version major|minor|patch
-
-        # Push branch and tags to repo
-        $ git push --follow-tags
-
-2. Preparing to publish to PyPi
+1 - Update `version.py`
 
-        # Bump the PyPi package to the same version
-        $ vi setup.py
+2 - Update `package.json`
 
-        # Commit to github
-        $ git add setup.py
-        $ git commit -m "Bump pypi package version to vx.x.x"
-
-3. Publish to npm and PyPi
+4 - Publish: `npm run publish-all` (Make sure you have access to NPM and PyPI)
+4b - If the `publish-all` script fails on the `twine` command, try running
+```sh
+twine upload dist/dash_core_components-xx.x.x.tar.gz # where xx.x.x is the version number
```

Comment:
@valentijnnieman I may be misunderstanding but was there a reason for the `xx` in the `xx.x.x` representation of the version? I would lean towards just using x.x.x to notate the version as I believe those digits will vary depending on what version we're on.